__tests__/unit/cloudstore.android.test.js

```diff
@@ -30,14 +30,11 @@ describe('CloudStore android unit test', () => {
 
     it('should not backup twice', async () => {
       const getItemSpy = jest.spyOn(GDriveCloudStorage, 'getItem').mockResolvedValue('resolved value');
-      const setItemSpy = jest.spyOn(GDriveCloudStorage, 'setItem');
+      const setItemSpy = jest.spyOn(GDriveCloudStorage, 'setItem').mockImplementation();
 
       await expect(CloudStore.putKey({ keyId, ciphertext })).rejects.toThrow(/already present/);
+      expect(setItemSpy.mock.calls.length).toBe(0);
       getItemSpy.mockRestore();
-
-      await CloudStore.putKey({ keyId, ciphertext });
-      expect(setItemSpy.mock.calls.length).toBe(2);
-
```

Comment:
Why remove this check here?

Reply:
I have restored these lines in a new commit.